packages/dcos-checks/extra/dcos-checks/cmd/checks.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	statusOK = iota
+	statusWarning
+	statusFailure
+	statusUnknown
+)
+
+// DCOSChecker defines an interface for a generic DC/OS check.
+// ID() returns a check unique ID and RunCheck(...) returns a combined stdout/stderr, exit code and error.
+type DCOSChecker interface {
+	ID() string
+	Run(context.Context, *CLIConfigFlags) (string, int, error)
```

Comment:
I know you commented on this above as to what the return values are, but we should probably name them in the interface definition.

```
Run(context.Context, *CLIConfigFlags) (output string, exitCode int, err error)
```